percona/watcher/wal.go

```diff
@@ -132,9 +131,13 @@ func getLatestBackup(ctx context.Context, cli client.Client, cr *pgv2.PerconaPGC
 	runningBackupExists := false
 	for _, backup := range backupList.Items {
 		backup := backup
-		if latest.Status.CompletedAt == nil || backup.Status.CompletedAt.After(latest.Status.CompletedAt.Time) {
-			latest = &backup
-		} else if backup.Status.State == pgv2.BackupStarting || backup.Status.State == pgv2.BackupRunning {
+		switch backup.Status.State {
+		case pgv2.BackupSucceeded:
+			if latest.Status.CompletedAt == nil || backup.Status.CompletedAt.After(latest.Status.CompletedAt.Time) {
```

Comment:
minor: as discussed in previous PRs, we can use `isZero` instead of `==nil` on `time` operations.